extensions/ext/GLSL_EXT_ray_flags_primitive_culling.txt

```diff
@@ -73,6 +74,40 @@ Modifications to the OpenGL Shading Language Specification, Version 4.60
       #define GL_EXT_ray_flags_primtive_culling          1
 
 
+Additions to Chapter 4 of the OpenGL Shading Language Specification
+(Variables and Types)
+
+    Modify Section 4.4, Layout Qualifiers
+
+    Modify the sentence that begins with "They can also appear with
+    just an interface-qualifier" to add "or even without one"
+
+    (Add additional line)
+    layout-qualifier;
+
+
+    Modify table in Section 4.4 (Layout Qualifiers)
+
+
+    Layout Qualifier    Qualifier  Individual   Block    Block      Allowed
+                          Only      Variable             Member    Interface
+    -----------------   ---------  ----------   ------   ------    ---------
+
+    primitive_culling      X                                         None
+
+
+    Add a new subsection to Section 4.4
+
+    4.4.10 Primitive Culling Layout Qualifier
+
+    This layout qualifier can only be applied at global scope
+    and without an interface-qualifier.
+
+    It is a compile-time error to use this qualifier with an
+    interface-qualifier, on an interface block or member of an interface
+    block, or with any other layout qualifiers.
+
```

Comment:
We should probably add something stating what this layout is actually *for*. 

Ie - to indicate that triangle or AABB primitive culling is being used and must be specified when using the SkipTriangles or SkipAABBs ray flags or ray flag constants?

Reply:
Updated